Head-of-Product.md

```diff
@@ -0,0 +1,62 @@
+## About [Snyk](https://snyk.io)
+
+<img src="https://snyk.io/images/snyk-dog.png" width="100" alt="Snyk logo" />
+
+Snyk's mission is to help developers use open source code and stay secure. 
+Open Source code, pulled from npm, Maven, RubyGems and others, is often the majority of code in an application, and yet it receives no security attention. 
+Most developers do nothing to track known vulnerabilities in the packages they use, and trust this code as though its their own. 
+This risk is massive today, and is only made worse by growing use of packages and trends such as Serverless and PaaS, who secure the server's dependencies but leave the app to fend with its dependencies security alone.
+
+At Snyk we believe this problem can only be fixed by developers and DevOps teams, and built into the way we create software. 
+We build a SaaS CI/CD and runtime monitoring product that helps organisations continuosly find and fix vulnerable dependencies without slowing down.
+
+It's important to understand Snyk is a dev tooling company at heart. 
+We care about securing the open source world itself as much as as we care about our enterprise grade solution that let large companies consume open source code while controlling risk. 
+We receive regular community contributions (and love!), have more than 600 badges in GitHub championing us, and continuously protect over 10,000 open source repositories. 
+
+Lastly, Snyk is a growth oriented venture backed company. We raised a $3M seed round from US investors in October '15, and received another large infusion from our investors recently, demonstrating their excitement about our progress and potential. 
+Our team numbers 17 amazing people in two large offices in Tel Aviv & London, and a smaller presence in the US. We take a lot of care to keep a warm and inclusive environment, and make sure our distributed team gets plenty of face time over video and in trips.
+
+## Head of Product - London or Israel
+
+We are looking for a senior head of product, based in one of our main offices in London or Tel Aviv. 
+
+In this role, you will own all product management activities, translating our vision and user feedback into product needs and working closely with engineering to make it a reality. You will also work regularly with customers and prospects, balancing high-touch enterprise customer interactions with understanding the many thousands of users who use your product self serve.
+
+As Snyk grows, you'll have the opportunity to build, lead and grow a product team, scaling up as our company hits its next growth stages. You will be a part of the company's leadership, and will influence practically all parts of the company. You will be reporting directly to the CEO.
+
+
+### Responsibilities
```

Comment:
Tough to make it non-generic... I think the previous pieces explain why it's special.